qa/suites/upgrade/nautilus-p2p/nautilus-p2p-stress-split/1-ceph-install/nautilus.yaml

```diff
@@ -1,11 +1,11 @@
 meta:
-- desc: install ceph/nautilus v14.2.2
+- desc: install ceph/nautilus v14.2.19
 tasks:
 - install:
-    tag: v14.2.2
+    tag: v14.2.19
```

Comment:
why not .20 here?

Reply:
I don't remember any related issues, perhaps, you are confusing it with https://tracker.ceph.com/issues/50595